python_modules/dagster/dagster/_core/definitions/run_request.py

```diff
@@ -100,7 +101,7 @@ class RunRequest(
         [
             ("run_key", PublicAttr[Optional[str]]),
             ("run_config", PublicAttr[Mapping[str, Any]]),
-            ("tags", PublicAttr[Mapping[str, str]]),
+            ("tags", PublicAttr[Mapping[str, Any]]),
```

Comment:
This one should still be `str` since validate tags normalizes everything to strings (your change to the `__new__` signature is correct though).

Reply:
The perks of trying to make python into a strictly typed lang 🙃